tests/app/celery/test_reporting_tasks.py

```diff
@@ -50,21 +50,31 @@ def test_create_nightly_billing_triggers_tasks_for_days(notify_api, mocker, day_
 
 
 @freeze_time('2019-08-01')
-@pytest.mark.parametrize('day_start, expected_kwargs', [
-    (None, ['2019-07-31', '2019-07-30', '2019-07-29', '2019-07-28']),
-    ('2019-07-21', ['2019-07-21', '2019-07-20', '2019-07-19', '2019-07-18']),
-])
-def test_create_nightly_notification_status_triggers_tasks_for_days(notify_api, mocker, day_start, expected_kwargs):
+def test_create_nightly_notification_status_triggers_tasks_for_days(notify_api, mocker):
     mock_celery = mocker.patch('app.celery.reporting_tasks.create_nightly_notification_status_for_day')
-    create_nightly_notification_status(day_start)
+    create_nightly_notification_status()
+
+    assert mock_celery.apply_async.call_count == (
+        (4 * 3)  # four days, three notification types
```

Comment:
Optional: Could change your logic to be `4 * 2 + 10` but not fussed if you don't